torch/utils/data/datapipes/utils/common.py

```diff
@@ -99,29 +99,41 @@ def deprecation_warning(name, new_name: str = ""):
 
 
 class StreamWrapper:
```

Comment:
I was planning to make `isinstance` check also working. But, after a while of research, this may not be possible because the `file_obj` can be arbitrary python file-like object. It's almost impossible for me to add a metaclass to handle all cases.